doc/source/user_guide/io.rst

```diff
@@ -3266,6 +3261,12 @@ clipboard (CTRL-C on many operating systems):
 
 And then import the data directly to a ``DataFrame`` by calling:
 
+.. ipython:: python
+   :suppress:
+
+   clipdf = pd.DataFrame({'A': [1, 2, 3], 'B': [4, 5, 6], 'C': ['p', 'q', 'r']},
+                         index=['x', 'y', 'z'])
```

Comment:
I'm not sure if this is really needed. Do you mind having a look @TanyaaCJain

Reply:
Oh yes.. I missed mentioning this. Thanks @datapythonista for the reminder! Even I couldn't find this assignment be of use anywhere in the file. So, should we go ahead with getting rid of this?

Reply:
I have he feeling that the `read_clipboard` was an `ipython` block before, and we had this `clipdf` and a `to_clipboard` before reading from the clipboard. But since not it's a python block and the code is written manually, I guess we can simply get rid of this.